qiskit/visualization/pulse_v2/interface.py

```diff
@@ -355,17 +353,17 @@ def draw(program: Union[Waveform, ParametricPulse, Schedule],
 
         You can partially customize a preset stylesheet when initializing it.
 
-        ```python
-            my_style = {
-                'formatter.channel_scaling.drive': 5,
-                'formatter.channel_scaling.control': 1,
-                'formatter.channel_scaling.measure': 5
-            }
-            style = IQXStandard(**my_style)
+            ```python
```

Comment:
This is markdown style syntax, please use a [code-block](https://www.sphinx-doc.org/en/1.5/markup/code.html#directive-code-block) instead.